denops_std/function/vim/prop_add_list.ts

```diff
@@ -6,29 +6,41 @@ export type PropAddListProps = {
   type: string;
 };
 
-export type PropAddListItem = [number, number, number, number];
+export type PropAddListItem = [
+  lnum: number,
+  col: number,
+  end_lnum: number,
+  end_col: number,
+];
```

Comment:
💭 

I didn't know that we can annotate each tuple elements. Could you share the URL for this spec with me if you have one (This is just my interest so just ignore if you don't have it)